dotcom-rendering/src/components/Doughnut.tsx

```diff
@@ -27,9 +27,10 @@ const SEGMENT_GAP = 2;
 const TAU = Math.PI * 2;
 const QUARTER_TURN = TAU / 4;
 
-const unitStyles = css`
+const percentageStyles = css`
 	${headlineBold34}
 	text-anchor: middle;
+	fill: var(--doughnut-percentage-icon);
```

Comment:
I think inheritance is superior, here!

```suggestion
	fill: currentColor;
```